src/Microsoft.ML.Data/Model/Onnx/SaveOnnxCommand.cs

```diff
@@ -40,11 +44,21 @@ public sealed class Arguments : DataCommand.ArgumentsBase
             [Argument(ArgumentType.AtMostOnce, HelpText = "Comma delimited list of input column names to drop", ShortName = "idrop", SortOrder = 5)]
             public string InputsToDrop;
 
-            [Argument(ArgumentType.AtMostOnce, HelpText = "Comma delimited list of output column names to drop", ShortName = "odrop", SortOrder = 6)]
+            [Argument(ArgumentType.AtMostOnce, Visibility = ArgumentAttribute.VisibilityType.EntryPointsOnly, HelpText = "Array of input column names to drop", SortOrder = 6)]
+            public string[] InputsToDropArray;
+
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Comma delimited list of output column names to drop", ShortName = "odrop", SortOrder = 7)]
             public string OutputsToDrop;
 
-            [Argument(ArgumentType.AtMostOnce, HelpText = "Whether we should attempt to load the predictor and attach the scorer to the pipeline if one is present.", ShortName = "pred", SortOrder = 7)]
+            [Argument(ArgumentType.AtMostOnce, Visibility = ArgumentAttribute.VisibilityType.EntryPointsOnly, HelpText = "Array of output column names to drop", SortOrder = 8)]
+            public string[] OutputsToDropArray;
+
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Whether we should attempt to load the predictor and attach the scorer to the pipeline if one is present.", ShortName = "pred", SortOrder = 9)]
             public bool? LoadPredictor;
```

Comment:
Is there a reason to have this in entry-point land?

Reply:
Not if we are passing in ITransformModel that contains the predictor.

---
In reply to: [191819078](https://github.com/dotnet/machinelearning/pull/248#discussion_r191819078) [](ancestors = 191819078)